src/jquery.daterangepicker.js

```diff
@@ -2393,18 +2393,6 @@
             return attrString;
         }
 
-        function daysFrom1970(t) {
-            return Math.floor(toLocalTimestamp(t) / 86400000);
-        }
-
-        function toLocalTimestamp(t) {

```

Comment:
It seems the use of this function allows for different 'types' of timestamps...

Reply:
Why? As long as `opt.start` and `opt.end` are timestamps, `moment(opt.end).diff(moment(opt.start), 'days')` should work fine.

Reply:
I am no JavaScript guru, but this is what Chrome Console give me when trying to run the index.html demonstration:

```
 Deprecation warning: value provided is not in a recognized ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non ISO date formats are discouraged and will be removed in an upcoming major release. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
Arguments: 
[0] _isAMomentObject: true, _isUTC: false, _useUTC: false, _l: undefined, _i: false, _f: undefined, _strict: undefined, _locale: [object Object]
x/<@https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.16.0/moment.min.js:23:668
qb@https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.16.0/moment.min.js:214:18
pb@https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.16.0/moment.min.js:212:274
ob@https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.16.0/moment.min.js:210:274
rb@https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.16.0/moment.min.js:217:160
sb@https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.16.0/moment.min.js:217:194
a@https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.16.0/moment.min.js:6:199
countDays@file:///tmp/jquery-date-range-picker/src/jquery.daterangepicker.js:1806:29
checkSelectionValid@file:///tmp/jquery-date-range-picker/src/jquery.daterangepicker.js:1723:24
setTime@file:///tmp/jquery-date-range-picker/src/jquery.daterangepicker.js:1400:13
renderTime@file:///tmp/jquery-date-range-picker/src/jquery.daterangepicker.js:1356:13
showTime@file:///tmp/jquery-date-range-picker/src/jquery.daterangepicker.js:2064:13
init_datepicker@file:///tmp/jquery-date-range-picker/src/jquery.daterangepicker.js:1036:21
$.fn.dateRangePicker@file:///tmp/jquery-date-range-picker/src/jquery.daterangepicker.js:944:9
@file:///tmp/jquery-date-range-picker/demo.js:95:2
i@https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js:2:27444
fireWith@https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js:2:28213
ready@https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js:2:30004
K@https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js:2:30366
```